sdk/iothub/azure-resourcemanager-iothub/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0 (2021-08-30)
 
 - Add support for new service API version 2021-07-01
     - Add flag for disabling local authentication to IotHubProperties
     - Add flags to disable device and/or module symmetric key based authentication to IotHubProperties
     - Add flag for restricting outbound network access to IotHubProperties
     - Add settable list of allowed fully qualified domain names for egress from IoT Hub to IotHubProperties
-
+- Upgraded `azure-core` dependency from `1.13.0` to `1.19.0` 
+  - [azure-core changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/CHANGELOG.md#1160-2021-05-07)
+- Upgraded `azure-core-management` dependency from `1.2.2` to `1.4.0` 
+  - [azure-core changelog](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-management/CHANGELOG.md#140-2021-08-06)
```

Comment:
You'd better to include changelog from release branch e.g. https://github.com/Azure/azure-sdk-for-java/tree/azure-core_1.19.0

The line number on main branch probably not going to stay the same.

Reply:
This link is tied to a header, not a line number. As long as the release notes header for the release isn't changed (which seems unlikely), it will work.